gflownet/proxy/base.py

```diff
@@ -2,8 +2,10 @@
 Base class of GFlowNet proxies
 """
 from abc import abstractmethod
+
 import numpy as np
 import numpy.typing as npt
+
 from gflownet.utils.common import set_device, set_float_precision
 
 
```

Comment:
```suggestion
class Proxy(ABC):
```
continuation of the previous suggestion

Reply:
I'm sorry, it got a bit messy but for some reason I couldn't add this suggestion it to the line below